cmd/cdi-apiserver/apiserver.go

```diff
@@ -0,0 +1,81 @@
+package main
```

Comment:
If this binary is only to support an uploader then I think the filename is too generic.

Reply:
@davidvossel added this but I think if fits with our conventions.  Is cdi-apiserver too generic?  If so, then that should prob change too.

Reply:
I pictured this component gaining additional functionality in the future.

For example, the KubeVirt apiserver handles our subresource endpoints and the CRD validation webhook as well.

Eventually i'd like the CDI apiserver to manage CRD validation too. Basically this component will be used for anything that extends the k8s apiserver for CDI specific functionality.